globus_sdk/auth/token_response.py

```diff
@@ -30,6 +33,7 @@ def __init__(self, *args, **kwargs):
         GlobusHTTPResponse.__init__(self, *args, **kwargs)
         # dict used to avoid re-computing self.by_resource_server
         self._by_resource_server = None
+        self._by_resource_server = self.by_resource_server
```

Comment:
This works, but I think it's confusing to read.

Can we just move the body of the `by_resource_server` property here, into `__init__`, and have the property just return the underlying value?
Or even just make `self.by_resource_server` an attribute on the object, and drop the `property` decorator altogether? (We'd need to move its description elsewhere in the docs, of course.)